python/src/runner.py

```diff
@@ -257,7 +257,12 @@ def prepareInteractive(reset=True):
     # clear the terminal, reset on Mac OSX and Linux. This prevents strange history bugs where
     # the command just entered is echoed again (2020-10-14).
     if reset:
-        os.system('cls' if os.name == 'nt' else 'reset')
+        if os.name == 'nt':
+            # clear the terminal
+            os.system('cls')
+        else: 
+            # On linux & mac use ANSI Sequence for this
+            print('\033[2J\033[H')
```

Comment:
@CodeSteak Gibt es hierfür einen Grund?

Reply:
`os.system('clear')` just prints out these characters (+ characters for clearing the buffer, which is often unsupported). It does feel strange to spawn a process for that.